src/foam/nanos/app/Health.c

```diff
@@ -0,0 +1,679 @@
+} foam_nanos_app_Health_t;
+
+const char *toSummary() {
+
+        return this->id + '/' + this->address;
+      
+}
+void init() {
+
+      var is = this->cls__getAxiomsByClass(foam->core->Import);
+      for ( var i = 0 ; i < strlen(is) ; i++ ) {
+        var imp = is[i];
+
+        if ( imp->required && ! this->__context__[imp->key + '$'] ) {
+          var m = 'Missing required import: ' + imp->key + ' in ' + this->cls_->id;
+          foam_assert(false, m);
+        }
+      }
+    
+}
+void initArgs(multitype_union_t args, multitype_union_t ctx) {
+
+      if ( ctx  ) this->__context__ = ctx;
+      if ( args ) this_copyFrom(args, true);
+    
+}
+void hasOwnProperty(multitype_union_t name) {
+
+      /**
+       * Returns true if this object is storing a value for a property
+       * named by the 'name' parameter->
+       */
+
+      return ! foam->Undefined_isInstance(this->instance_[name]);
+    
+}
+void hasDefaultValue(multitype_union_t name) {
+
+      if ( ! this_hasOwnProperty(name) ) return true;
+
+      var axiom = this->cls__getAxiomByName(name);
+      return axiom_isDefaultValue(this[name]);
+    
+}
+void clearProperty(multitype_union_t name) {
+
+      /**
+       * Undefine a Property's value->
+       * The value will revert to either the Property's 'value' or
+       * 'expression' value, if they're defined or undefined if they aren't->
+       * A propertyChange event will be fired, even if the value doesn't change->
+       */
+
+      var prop = this->cls__getAxiomByName(name);
+      foam_assert(prop && foam->core->Property_isInstance(prop),
+        'Attempted to clear non-property', name);
+
+      if ( this_hasOwnProperty(name) ) {
+        var oldValue = this[name];
+        this->instance_[name] = undefined;
+        this_clearPrivate_(name);
+
+        // Avoid creating slot and publishing event if nobody is listening->
+        if ( this_hasListeners('propertyChange', name) ) {
+          this_pub('propertyChange', name, this_slot(name));
+        }
+      }
+    
+}
+void setPrivate_(multitype_union_t name, multitype_union_t value) {
+
+      /**
+       * Private support is used to store per-object values that are not
+       * instance variables->  Things like listeners and topics->
+       */
+      ( this->private_ || ( this->private_ = {} ) )[name] = value;
+      return value;
+    
+}
+void getPrivate_(multitype_union_t name) {
+
+      return this->private_ && this->private_[name];
+    
+}
+void hasOwnPrivate_(multitype_union_t name) {
+
+      return this->private_ && ! foam->Undefined_isInstance(this->private_[name]);
+    
+}
+void clearPrivate_(multitype_union_t name) {
+
+      if ( this->private_ ) this->private_[name] = undefined;
+    
+}
+void createListenerList_() {
+
+      /**
+       * This structure represents the head of a doubly-linked list of
+       * listeners-> It contains 'next', a pointer to the first listener,
+       * and 'children', a map of sub-topic chains->
```

Comment:
Unfortunately the english language doesn't use pointers in order to denote end of sentences
```suggestion
       * and 'children', a map of sub-topic chains.
```